doc/releases/changelog-0.25.0.md

```diff
@@ -615,6 +612,11 @@ of operators.
   supported by `qml.qchem`.
   [(#2795)](https://github.com/PennyLaneAI/pennylane/pull/2795)
```

Comment:
That is correct for some functions: changing `qml.hf` to `qml.qchem` is enough. But some of the function names were modified to make them compatible with the old PL-QChem. It seems better to me that we just refer the users to the `qml.qchem` documentations and demos instead of providing a simple recipe.

Reply:
Thanks Soran! Would be good to add what you say above to the changelog entry

Reply:
I'll stick this in the changelog entry :)